2019-01-31 22:10:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Unknown data type: "("; SQL statement:
CREATE TABLE IF NOT EXISTS PORT_INVENTORIES(ID             INTEGER PRIMARY KEY NOT NULL,PORT_ID        INTEGER             NOT NULL,COMMODITY_ID   INTEGER             NOT NULL,ON_HAND        INTEGER             NOT NULL,BUY_PRICE      INTEGER             NOT NULL,SELL_PRICE     INTEGER             NOT NULL,FOREIGN_KEY (PORT_ID) REFERENCES PORTS(ID),FOREIGN_KEY (COMMODITY_ID) REFERENCES COMMODITIES(ID)); [50004-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4400)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4215)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:6662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6570)
	at org.h2.command.Parser.parseCreate(Parser.java:4615)
	at org.h2.command.Parser.parsePrepared(Parser.java:380)
	at org.h2.command.Parser.parse(Parser.java:335)
	at org.h2.command.Parser.parse(Parser.java:307)
	at org.h2.command.Parser.prepareCommand(Parser.java:278)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at TMP.PortInventory.createTable(PortInventory.java:56)
	at test.basictest.main(basictest.java:39)
2019-01-31 22:10:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
MERGE INTO PORT_INVENTORIES(ID,PORT_ID,COMMODITY_ID,ON_HAND, BUY_PRICE, SELL_PRICE) KEY(ID) VALUES(0,0,1,10,15,100); [42102-197]
2019-01-31 22:10:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
MERGE INTO PORT_INVENTORIES(ID,PORT_ID,COMMODITY_ID,ON_HAND, BUY_PRICE, SELL_PRICE) KEY(ID) VALUES(1,0,2,1,222,101); [42102-197]
2019-01-31 22:10:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
MERGE INTO PORT_INVENTORIES(ID,PORT_ID,COMMODITY_ID,ON_HAND, BUY_PRICE, SELL_PRICE) KEY(ID) VALUES(2,1,0,122,0,0); [42102-197]
2019-01-31 22:10:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
MERGE INTO PORT_INVENTORIES(ID,PORT_ID,COMMODITY_ID,ON_HAND, BUY_PRICE, SELL_PRICE) KEY(ID) VALUES(3,2,2,1,2910,6); [42102-197]
2019-01-31 22:10:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
SELECT * FROM PORT_INVENTORIES; [42102-197]
2019-01-31 22:11:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Unknown data type: "("; SQL statement:
CREATE TABLE IF NOT EXISTS PORT_INVENTORIES(ID             INTEGER PRIMARY KEY NOT NULL,PORT_ID        INTEGER             NOT NULL,COMMODITY_ID   INTEGER             NOT NULL,ON_HAND        INTEGER             NOT NULL,BUY_PRICE      INTEGER             NOT NULL,SELL_PRICE     INTEGER             NOT NULL,FOREIGN_KEY(PORT_ID) REFERENCES PORTS(ID),FOREIGN_KEY(COMMODITY_ID) REFERENCES COMMODITIES(ID)); [50004-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4400)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4215)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:6662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6570)
	at org.h2.command.Parser.parseCreate(Parser.java:4615)
	at org.h2.command.Parser.parsePrepared(Parser.java:380)
	at org.h2.command.Parser.parse(Parser.java:335)
	at org.h2.command.Parser.parse(Parser.java:307)
	at org.h2.command.Parser.prepareCommand(Parser.java:278)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at TMP.PortInventory.createTable(PortInventory.java:56)
	at test.basictest.main(basictest.java:39)
2019-01-31 22:11:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
MERGE INTO PORT_INVENTORIES(ID,PORT_ID,COMMODITY_ID,ON_HAND, BUY_PRICE, SELL_PRICE) KEY(ID) VALUES(0,0,1,10,15,100); [42102-197]
2019-01-31 22:11:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
MERGE INTO PORT_INVENTORIES(ID,PORT_ID,COMMODITY_ID,ON_HAND, BUY_PRICE, SELL_PRICE) KEY(ID) VALUES(1,0,2,1,222,101); [42102-197]
2019-01-31 22:11:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
MERGE INTO PORT_INVENTORIES(ID,PORT_ID,COMMODITY_ID,ON_HAND, BUY_PRICE, SELL_PRICE) KEY(ID) VALUES(2,1,0,122,0,0); [42102-197]
2019-01-31 22:11:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
MERGE INTO PORT_INVENTORIES(ID,PORT_ID,COMMODITY_ID,ON_HAND, BUY_PRICE, SELL_PRICE) KEY(ID) VALUES(3,2,2,1,2910,6); [42102-197]
2019-01-31 22:11:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PORT_INVENTORIES" not found; SQL statement:
SELECT * FROM PORT_INVENTORIES; [42102-197]
2019-01-31 22:51:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "PORT_ID" not found [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3148)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3247)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:346)
	at TMP.Route.retrieveAll(Route.java:131)
	at test.basictest.main(basictest.java:78)
2019-01-31 23:16:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "PORT_ID" not found [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3148)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3247)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:346)
	at TMP.Route.retrieveRouteCosts(Route.java:163)
	at test.basictest.main(basictest.java:91)
2019-01-31 23:52:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "MIKE" not found; SQL statement:
MERGE INTO MERCHANTS(ID,NAME,HOME_PORT,CURRENT_PORT,CAPACITY) VALUES(0,Mike,0,0,CAPACITY); [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Merge.prepare(Merge.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at TMP.Merchant.store(Merchant.java:213)
	at test.basictest.main(basictest.java:80)
2019-01-31 23:52:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "NIAMH" not found; SQL statement:
MERGE INTO MERCHANTS(ID,NAME,HOME_PORT,CURRENT_PORT,CAPACITY) VALUES(1,Niamh,1,0,CAPACITY); [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Merge.prepare(Merge.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at TMP.Merchant.store(Merchant.java:213)
	at test.basictest.main(basictest.java:80)
2019-01-31 23:52:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "CORTEZ" not found; SQL statement:
MERGE INTO MERCHANTS(ID,NAME,HOME_PORT,CURRENT_PORT,CAPACITY) VALUES(2,Cortez,1,2,CAPACITY); [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Merge.prepare(Merge.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at TMP.Merchant.store(Merchant.java:213)
	at test.basictest.main(basictest.java:80)
2019-01-31 23:52:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "CAPACITY" not found; SQL statement:
MERGE INTO MERCHANTS(ID,NAME,HOME_PORT,CURRENT_PORT,CAPACITY) VALUES(0,'Mike',0,0,CAPACITY); [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Merge.prepare(Merge.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at TMP.Merchant.store(Merchant.java:213)
	at test.basictest.main(basictest.java:80)
2019-01-31 23:52:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "CAPACITY" not found; SQL statement:
MERGE INTO MERCHANTS(ID,NAME,HOME_PORT,CURRENT_PORT,CAPACITY) VALUES(1,'Niamh',1,0,CAPACITY); [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Merge.prepare(Merge.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at TMP.Merchant.store(Merchant.java:213)
	at test.basictest.main(basictest.java:80)
2019-01-31 23:52:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "CAPACITY" not found; SQL statement:
MERGE INTO MERCHANTS(ID,NAME,HOME_PORT,CURRENT_PORT,CAPACITY) VALUES(2,'Cortez',1,2,CAPACITY); [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Merge.prepare(Merge.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)
	at TMP.Merchant.store(Merchant.java:213)
	at test.basictest.main(basictest.java:80)
2019-02-01 18:41:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS MERCHANTS( ID INTEGER PRIMARY KEY NOT NULL, NAME TEXT NOT NULL, HOME_PORT INTEGER NOT NULL, CURRENT_PORT INTEGER NOT NULL, CAPACITY INTEGER NOT NULL, FOREIGN KEY(HOME_PORT) REFERENCES PORTS (ID), FOREIGN KEY(CURRENT_PORT) REFERENCES PORTS (ID);[*]"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE IF NOT EXISTS MERCHANTS( ID INTEGER PRIMARY KEY NOT NULL, NAME TEXT NOT NULL, HOME_PORT INTEGER NOT NULL, CURRENT_PORT INTEGER NOT NULL, CAPACITY INTEGER NOT NULL, FOREIGN KEY(HOME_PORT) REFERENCES PORTS (ID), FOREIGN KEY(CURRENT_PORT) REFERENCES PORTS (ID); [42001-196]
